### ISSUE TITLE
etbCounter needs to be Removed in the Future

### ISSUE THREAD

**Issue by Hanmac** [Card-Forge/forge#210]
etbCounter currently makes problems with CardTextChanges because the value for X is not affected by it but it should.
Example: `Ulasht, the Hate Seed`

In the work of changing ETB Keywords into a Form that used "ReplacementResult.Updated" instead of using ETBInternal,
it will be much cleaner to use normal replacement Effects instead of using a Keyword

This is part of the Cleanup of CardFactoryUtil to remove everything what isn't a "real Keyword"

**Comment by Northmoc** [comment]
Just some cards that I think might need special attention when doing this refactor:
 - Agent's Toolkit
 - the IKO "this card enters with your choice of counter A or counter B" stuff

**Comment by Hanmac** [comment]
yeah, extracting from a keyword into an replacement effect isn't that big of a problem.

it's just to not use the keyword anymore

**Comment by Hanmac** [comment]
@tool4ever if we do not remove this keyword, then maybe you can do your thing with #5666 with these Keywords too so it would fix `Ulasht, the Hate Seed`?

**Comment by tool4ever** [comment]
I don't know, there's no benefit in keeping it? (except saving some editing work?)

**Comment by Hanmac** [comment]
> I don't know, there's no benefit in keeping it? (except saving some editing work?)

I don't think so ... 

Having fake keywords (like this) or hidden keywords only causes Problems Rules-wise ... 

So i'm for replacing them with Replacement Effects when we need to touch them anyway

**Comment by Hanmac** [comment]
@dracontes should we do another round with etbCounter removal? Or do we wait with this after a Bloomborrow release?

**Comment by dracontes** [comment]
> should we do another round with etbCounter removal? Or do we wait with this after a Bloomborrow release?

My apologies for the late reply. I'm fine with the `etbCounter` removal pass being after a Bloomburrow release as I'm currently organizing my notes and going over minor items in them.
